low-level/sspl-ll.spec

```diff
@@ -63,17 +63,23 @@ SSPL_BASE=${RPM_BUILD_ROOT}/opt/seagate/%{product_family}/sspl
 mkdir -p $SSPL_BASE
 mkdir -p ${RPM_BUILD_ROOT}/etc/{systemd/system,dbus-1/system.d,polkit-1/rules.d,sspl-ll/templates/snmp}
 cp -afv files/etc ${RPM_BUILD_ROOT}/
-cp -afv files/opt/seagate/sspl/* $SSPL_BASE/
-
-# Rename setup directory to bin directory and remove .py extension of sspl_setup files.
-mv $SSPL_BASE/setup $SSPL_BASE/bin
-mv $SSPL_BASE/bin/sspl_setup.py $SSPL_BASE/bin/sspl_setup
+cp -afv files/opt/seagate/sspl/conf $SSPL_BASE/
+mkdir -p $SSPL_BASE/bin
 
 # Copy the service into /opt/seagate/%{product_family}/sspl where it will execute from
 cp -rp __init__.py $SSPL_BASE
 mkdir -p $SSPL_BASE/low-level
 cp -rp . $SSPL_BASE/low-level
 
+# Coping independent executable script inside sspl/low-level to easier access core code access.
+SSPL_SETUP=$SSPL_BASE/low-level/files/opt/seagate/sspl/setup
+cp -p $SSPL_SETUP/generate_resource_health_view/resource_health_view $SSPL_BASE/low-level/
+cp -p $SSPL_SETUP/generate_sspl_bundle/sspl_bundle_generate $SSPL_BASE/low-level/
```

Comment:
support bundle & manifest generation scripts should be available for invocation independently too (as per recent discussion in DR review meeting), beyond support bundle interfaces. So better to keep them under /opt/seagate/sspl/bin again

Reply:
It is /opt/seagate/sspl/bin only. we are copying that independent file to low-level to access SSPL code and then we are moving that to sspl/bin.
Please see the below comments:
https://github.com/Seagate/cortx-monitor/pull/304#discussion_r573594603